Onboarding/QA/intro-and-welcome.md

```diff
@@ -1,795 +0,0 @@
-  button_labels = map(message.buttons, & &1.value.title)
-
-  buttons(SelectNextJourney: "@button_labels[0]") do
-    text("@message_text")
-  end
-end
-
-card DataPreferencesSelectedError, then: DataPreferencesSelectedError do
-  buttons(SelectNextJourney: "@button_labels[0]") do
-    text("@button_error_text")
-  end
-end
-
-card SelectNextJourney when contact.intent == "create profile" do
-  # Go to Profile Classifier journey
-  log("Navigating to Profile Classifier")
-  write_result("intro_completed", "yes")
-  run_stack("bd590c1e-7a06-49ed-b3a1-623cf94e8644")
-end
-
-card SelectNextJourney when contact.intent == "explore" do
-  # Go to Explore journey
-  log("Navigating to Explore")
-  write_result("intro_completed", "yes")
-  run_stack("4288d6a9-23c9-4fc6-95b7-c675a6254ea5")
-end
-
-card SelectNextJourney do
-  # TODO: How did we get here and what should we do in this case?
-  log("Unknown intent @intent. User stuck.")
-  write_result("intro_completed", "yes")
-end
-
-```
-
-## TODO
-
-Temporary TODO card to route to when we haven't created the destination yet
-
-```stack
-card TODO do
-  search =
-    get(
-      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
-      query: [
-        ["whatsapp", "true"],
-        ["slug", "todo"]
-      ],
-      headers: [["Authorization", "Token @global.config.contentrepo_token"]]
-    )
-
-  page_id = search.body.results[0].id
-
-  page =
-    get(
-      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
-      query: [
-        ["whatsapp", "true"]
-      ],
-      headers: [["Authorization", "Token @global.config.contentrepo_token"]]
-    )
-
-  text("@page.body.body.text.value.message")
-end
-
-```
-
-## Content dependancies
-
-Content is stored in the content repo, and referenced in the stack by slug. This means that we require the following slugs to be present in the contentrepo, and we're making the following assumptions:
-
-* `mnch_onboarding_welcome` , whatsapp message with two buttons
-* `mnch_onboarding_languages` , whatsapp message with 6 options.
-* `mnch_onboarding_language-updated`, whatsapp message with two buttons, variable `{language selection}` in message content will be replaced with selected language
-* `mnch_onboarding_pp_document`, whatsapp message with three buttons, and a document
-* `mnch_onboarding_pp_not_accepted`, whatsapp message with two buttons
-* `mnch_onboarding_pp_summary`, whatsapp message with two buttons, and a document
-* `mnch_onboarding_opt_in`, whatsapp message with two buttons
-* `mnch_onboarding_intent`, whatsapp message with 7-item list (hardcoded for now)
-* `mnch_onboarding_data_preferences`, whatsapp message with 3 buttons
-* `mnch_onboarding_data_preferences_yes`, whatsapp message
-
-## Error messages
-
-* `mnch_onboarding_error_handling_button`, for when a user sends in a message when we're expecting them to press one of the buttons
```

Comment:
It seems a bit odd that we're just removing these entire flows, maybe I'm missing something?

Reply:
They're in the wrong folder - I forgot to delete them when I copied them over to the /flows directory.